src/components/Card.js

```diff
@@ -37,12 +62,18 @@ export default class Card {
   }
 
   getView() {
-    this._cardElement = document
-      .querySelector(this._cardSelector)
-      .content.querySelector(".card")
+    const cardTemplate = document.querySelector(this._cardSelector);
+    if (!cardTemplate) {
+      console.error(`Card template selector ${this._cardSelector} not found.`);
+      return null;
+    }
+
+    this._cardElement = cardTemplate.content
+      .querySelector(".card")
       .cloneNode(true);
     const cardImageEl = this._cardElement.querySelector("#card-image");
```

Comment:
**Could be improved**

All elements found **every time** you call a method should be defined as **class fields** in the constructor, `_setEventListeners` (or in `getView`) only **1 time** in order not to waste resources on searching them again and again.

`Class fields` (the so-called **class variables** with `this`) are accessible inside any class method. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.
There can be `this._inputs`, `this._submitButton`, `this._cardImage`, `this._form`, `this._likeButton`, and so on.

```js
  getView() {
    this._element = this._getTemplate();
    this._likeButton = this._element.querySelector(".card__like-button");
    this._cardImage = this._element.querySelector(".card__image");
    this._trashButton = this._element.querySelector(".card__trash-button");

```